GuessTheWord-Starter/app/src/main/java/com/example/android/guesstheword/screens/score/ScoreFragment.kt

```diff
@@ -40,17 +40,17 @@ class ScoreFragment : Fragment() {
     private lateinit var viewModelFactory: ScoreViewModelFactory
 
     override fun onCreateView(
```

Comment:
Can you try to add all parameters on a single line? If needed, please change the line length to 120

Reply:
Sure, will do. I'm just wondering though if 120 columns won't be too wide for the side-by-side diff view, for example? More horizontal scrolling will be needed to see all the code in each column. Do you always work with 120?

Reply:
Not really. It depends on the Team's choices